core/key.go

```diff
@@ -401,6 +421,28 @@ func (k CollectionSchemaVersionKey) ToDS() ds.Key {
 	return ds.NewKey(k.ToString())
 }
 
+func (k SchemaHistoryKey) ToString() string {
+	result := COLLECTION_SCHEMA_VERSION + "/" + string(PriorityKey)
```

Comment:
question: Same here as the above `value` suffix.

Reply:
note: Discussion and resolution will be found against the other comment and applied here.